docs/licencjat.tex

```diff
@@ -246,6 +246,21 @@ \chapter{Conclusions}
 \chapter{Team members contribution}
 	\krystynaintodo{Podział prac}
 
+\chapter{Dictionary}
+    \begin{itemize}
+        \item Neural network - a machine learning model, inspired by a structure of a human brain. It consits of a large collection of neurons grouped into layers. Each neuron is connected to a set of other neurons. Each connection has an assigned value (also known as weight), which measures its importance.
+
+        \item Training - a process which consists of modification of connection weights after processing input data by a neural network and comparing its output with the desired output.
+
+        \item Inference - a process of computing neural network output based on the input and values of neuron weights
+
+        \item Convolution (discrete) - a mathematical function defined as follows
```

Comment:
Napisz najpierw o zwykłej konwolucji, a potem o tej dyskretnej. Dobry pomysł z umieszczeniem tego w słowniczku, można usunąć TODO z tłumaczeniem co to jest dyskretna konwolucja.